dune/gdt/local/operators/advection-dg.hh

```diff
@@ -192,10 +195,12 @@ public:
     outside_local_dofs_.resize(outside_basis.size(param));
     inside_local_dofs_ *= 0.;
     outside_local_dofs_ *= 0.;
+    const auto local_flux = numerical_flux_->flux().local_function();
+    local_flux->bind(intersection.inside());
```

Comment:
s.a., can be member, need inside/outside

Reply:
Inside/outside done, will do the member thing in the other PR to avoid doing the work twice.